website/docs/fundamentals/getting_started.mdx

```diff
@@ -235,6 +235,13 @@ This will print "Hello world" in the console.
 </TabItem>
 </Tabs>
 
+## Going further: Installing code snippets
+
+If you are using `Flutter` and `Vscode` consider [Flutter Riverpod Snippets](https://marketplace.visualstudio.com/items?itemName=robertbrunhage.flutter-riverpod-snippets)
+
+![img](../gifs/greetingProvider.gif)
```

Comment:
the files should be added in `/static/img`, then the path changed to `/img/...`

Otherwise they won't work
<img width="431" alt="Screenshot 2020-07-03 at 06 33 17" src="https://user-images.githubusercontent.com/20165741/86435188-53f91c00-bcf7-11ea-85cc-d1f40950cdb1.png">


You can test it locally by running `npm install` & `npm start` inside `website.
Alternatively you can use Netlify's preview link in the pull request